test/e2e/utils/kubernetes.go

```diff
@@ -466,6 +467,16 @@ func CheckExternalAddress(c HTTPClient, hostAndPort string) bool {
 	return resp.StatusCode == http.StatusOK
 }
 
+func isTlsHandskakeTimeoutError(err error) bool {
+	if err != nil {
+		if urlError, ok := err.(*url.Error); ok {
```

Comment:
new linter will fails with 
`test/e2e/utils/kubernetes.go:472:22: type assertion on error will fail on wrapped errors. Use errors.As to check for specific errors (errorlint)`